packages/material/scss/dialog/_layout.scss

```diff
@@ -1,27 +1,50 @@
 @import "~@progress/kendo-theme-default/scss/dialog/layout";
 
 @include exports("dialog/layout/material") {
+
+    $dialog-button-padding-x: 8px !default;
+    $dialog-button-padding-y: $button-padding-y !default;
+
+    .k-dialog {}
+
+    .k-dialog-content.k-scroll {
+        border-width: 1px 0;
+        border-style: solid;
+    }
+
     .k-dialog-buttongroup {
-        display: flex;
-        flex-wrap: nowrap;
+        padding: $dialog-button-padding-x;
+        border-width: 0;
         justify-content: flex-end;
-        align-items: center;
-        align-content: flex-end;
         box-shadow: none;
-        border-width: 0;
-        padding: $window-content-padding;
 
         .k-button {
-            border-radius: $border-radius;
-            flex: 0 1 auto;
-            align-self: center;
-            padding: 0 $padding-x;
-            border-width: 0;
-            transition: none;
-
-            &:first-child {
-                margin-right: $window-content-padding;
+            @extend .k-button.k-flat;
```

Comment:
After the material/button branch, we can now `@extend` the selectors to have flat buttons